frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -80,12 +80,12 @@ const GitSection: React.FC<GitSectionProps> = ({
       showGitType && setFieldTouched('git.type', false);
 
       const gitService = getGitService({ url, ref, contextDir: dir }, gitType as GitProvider);
-      const isReachable = gitService && (await gitService.isRepoReachable());
+      const status = gitService && (await gitService.isRepoReachable());
 
-      setIsRepoReachable(isReachable);
+      setRepoStatus(status);
       setFieldValue('git.isUrlValidating', false);
 
-      if (!isReachable) {
+      if (status !== RepoStatus.Reachable) {
```

Comment:
```suggestion
      if (repoStatus !== RepoStatus.Reachable) {
```

Reply:
Thanks. `repoStatus` was already used so I changed it to `repositoryStatus`.